src/test/java/uk/gov/hmcts/reform/prl/services/c100respondentsolicitor/C100RespondentSolicitorServiceTest.java

```diff
@@ -539,112 +540,17 @@ public void setUp() {
         when(systemUserService.getSysUserToken()).thenReturn("");
     }
 
-    @Test
-    public void populateAboutToStartCaseDataResSolConsentingToApplicationTest() {
-
-        callbackRequest.setEventId("c100ResSolConsentingToApplicationA");
-        Map<String, Object> response = respondentSolicitorService.populateAboutToStartCaseData(
-            callbackRequest
-        );
-
-        assertTrue(response.containsKey("respondents"));
-    }
-
-    @Test
-    public void populateAboutToStartCaseDataForResSolKeepDetailsPrivateTest() {
-        callbackRequest.setEventId("c100ResSolKeepDetailsPrivateA");
```

Comment:
Not enough test coverage in this file. it is now at 50%. It will fail once we merge.

Reply:
Resolved